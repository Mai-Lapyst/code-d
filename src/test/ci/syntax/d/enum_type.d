uint BIT = 1;
enum uint BIT = 1;
enum ABCD BIT = 1;