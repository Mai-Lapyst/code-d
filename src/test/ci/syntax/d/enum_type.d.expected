[{"start":0,"end":4,"scope":"storage.type.basic-type.d"},{"start":4,"end":5,"scope":"source.d"},{"start":5,"end":8,"scope":"variable.d"},{"start":8,"end":9,"scope":"source.d"},{"start":9,"end":10,"scope":"keyword.operator.assign.d"},{"start":10,"end":11,"scope":"source.d"},{"start":11,"end":12,"scope":"constant.numeric.integer.decimal.d"},{"start":12,"end":14,"scope":"meta.statement.end.d"}]
[{"start":0,"end":4,"scope":"storage.type.enum.d"},{"start":4,"end":5,"scope":"source.d"},{"start":5,"end":9,"scope":"storage.type.basic-type.d"},{"start":9,"end":10,"scope":"source.d"},{"start":10,"end":13,"scope":"entity.name.type.enum.d"},{"start":13,"end":14,"scope":"source.d"},{"start":14,"end":15,"scope":"keyword.operator.assign.d"},{"start":15,"end":16,"scope":"source.d"},{"start":16,"end":17,"scope":"constant.numeric.integer.decimal.d"},{"start":17,"end":19,"scope":"meta.statement.end.d"}]
[{"start":0,"end":4,"scope":"storage.type.enum.d"},{"start":4,"end":5,"scope":"source.d"},{"start":5,"end":9,"scope":"entity.name.type.d"},{"start":9,"end":10,"scope":"source.d"},{"start":10,"end":13,"scope":"entity.name.type.enum.d"},{"start":13,"end":14,"scope":"source.d"},{"start":14,"end":15,"scope":"keyword.operator.assign.d"},{"start":15,"end":16,"scope":"source.d"},{"start":16,"end":17,"scope":"constant.numeric.integer.decimal.d"},{"start":17,"end":19,"scope":"meta.statement.end.d"}]